feedparser/parsers/json.py

```diff
@@ -84,15 +84,19 @@ def parse_entry(self, e):
         for src, dst in self.ITEM_FIELDS:
             if src in e:
                 entry[dst] = e[src]
-
+        contents = []
```

Comment:
another thing I noticed.. there is no reason to skip the html content if both text and html contents exists. I think it's also better to keep the same structure (list with FeedParserDicts) as in the StrictParser.

@kurtmckee I may have missed something here though... In the StrictParser return value I see also `language` and `base` keys which are not present here.. Is it fine to omit them in general case?